src/app/pages/channels/channels.component.ts

```diff
@@ -112,6 +112,7 @@ export class ChannelsComponent implements OnInit {
     // close create row
     event.confirm.resolve();
 
+    event.newData.type && (event.newData.metadata = {'type': event.newData.type});
```

Comment:
It would be more readable to use
```
const type = event.newData.type;
if (type) {
    event.newData.metadata = {'type': type});
}
```

Reply:
This is a javascript/typescript idiom. Plus, we don't use redundant declaration/initialization.